web-bluetooth/link-loss.js

```diff
@@ -0,0 +1,70 @@
+var alertLevelCharacteristic;
+
+function onReadButtonClick() {
+  log('Requesting Bluetooth Device...');
+  navigator.bluetooth.requestDevice(
+    {filters: anyDevice(), optionalServices: ['link_loss']})
+  .then(device => {
+    log('Connecting to GATT Server...');
+    return device.gatt.connect();
+  })
+  .then(server => {
+    log('Getting Link Loss Service...');
+    return server.getPrimaryService('link_loss');
+  })
+  .then(service => {
+    log('Getting Alert Level Characteristic...');
+    return service.getCharacteristic('alert_level');
+  })
+  .then(characteristic => {
+    alertLevelCharacteristic = characteristic;
+    document.querySelector('#writeButton').disabled = false;
+    log('Reading Alert Level...');
+    return characteristic.readValue();
+  })
+  .then(value => {
+    log('> Alert Level changed to: ' + getAlertLevel(value));
```

Comment:
1) I prefer keeping the read value in the `<pre>` log. I've added however a "Choose" default element.
2) My bad, I've changed wrong string. `'> Alert Level changed to: ` is only when writing new value to the Alert Level characteristic.